cogs/help.py

```diff
@@ -228,6 +229,13 @@ def __init__(self, bot: commands.Bot):
             HelpButton("Nextcord", style=ButtonStyle.blurple, custom_id="nextcord")
         )
         self.add_item(HelpButton("Python", style=ButtonStyle.green, custom_id="python"))
+    
+    async def interaction_check(self, interaction: Interaction):
+        if interaction.user.get_role(883649900913512508) is not None:
```

Comment:
Unsure if this could be done like the others with a constant at the top of this file